packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/mira/MiraController.java

```diff
@@ -269,47 +271,71 @@ public ResponseEntity<JsonNode> generateModelLatex(@RequestBody final JsonNode m
 		}
 	)
 	public ResponseEntity<JsonNode> latexToAMR(@RequestBody final String latex) {
-		// create request:
-		final TaskRequest req = new TaskRequest();
-		req.setType(TaskType.MIRA);
+		////////////////////////////////////////////////////////////////////////////////
+		// 1. Convert latex string to python sympy code string
+		//
+		// Note this is a gollm string => string task
+		////////////////////////////////////////////////////////////////////////////////
```

Comment:
Because it has been decided to split the request between MIRA and GoLLM instead of just one task-runner. This should be in the Knowledge Controller, not in the MIRA one.